webapp/src/components/general/streaming/movie/Player.tsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
-import icon from '../../../../../assets/icons/play.png';
 import '../../../../../public/assets/css/player.css';
+import { Play } from 'react-feather';
 
 function Player() {
   return (
     <button className={'play'}>
```

Comment:
pas besoin de { pour les strings.
 
```suggestion
    <button className='play'>
```